build-scripts/profile_tool.py

```diff
@@ -9,10 +9,17 @@
 import os.path
 import sys
 
-import ssg.build_profile
-import ssg.constants
-import ssg.xml
-import ssg.build_yaml
+try:
+    import ssg.build_profile
+    import ssg.constants
+    import ssg.xml
+    import ssg.build_yaml
+except ImportError:
+    print("The ssg module could not be found.")
+    print("Run .pyenv.sh available in the project root diretory,"
+          "or add it to PYTHONPATH manually.")
```

Comment:
```suggestion
          " or add it to PYTHONPATH manually.")
```
Just aesthetic nitpick... I'll commit through github